src/test/crimson/test_crimson_scrub.cc

```diff
@@ -1124,7 +1124,7 @@ TEST_P(TestSnapSetCloneError, CloneError) {
       auto to_insert = make_clone(name, should_exist[i]);
       if (GetParam().should_inject_size(i)) {
 	expected_error.set_size_mismatch();
-	to_insert.second = so_builder_t(to_insert.second).set_size(
+	to_insert.second = so_builder_t{to_insert.second}.set_size(
```

Comment:
ditto

Reply:
same - won't make the CI tests otherwise.